text.md

```diff
@@ -0,0 +1 @@
+This file  should be ignored by git
```

Comment:
- Kindly remove the project's unrelated files from your repo. Follow the best coding practice to keep your code clean.